src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -528,6 +923,63 @@ private void checkPagination(boolean descending, int docCount, int docsPerPage,
     }
 
     /**
+     * For use with multi-value key view tests.
+     *
+     * Helper to test paging the view with multiple changes of direction. Uses a pre-defined
+     * array of queries for multi-value key testing.
+     *
+     * @param descending  true if the view is in descending order, and false otherwise.
+     * @param docCount    the total number of documents in the view.
+     * @param docsPerPage the maximum number of documents per page in the view.
+     * @param useNewView  true for creating a new view for the query, and false otherwise.
+     * @param pageToPages the list of page numbers to page to.
+     */
+    private void checkPaginationWithMultiValueKey(boolean descending, boolean useNewView,
+                                                  int docCount, int docsPerPage,
+                                                  int... pageToPages) {
```

Comment:
This shouldn't be a vararg method, it makes things very hard to follow at call sites.